cornflow/endpoints/execution.py

```diff
@@ -60,26 +59,33 @@ def post(self):
           the reference_id for the newly created execution if successful) and a integer wit the HTTP status code
         :rtype: Tuple(dict, integer)
         """
+        config = current_app.config
+        airflow_conf = dict(url=config['AIRFLOW_URL'], user=config['AIRFLOW_USER'], pwd=config['AIRFLOW_PWD'])
+
         self.user_id, self.admin, self.super_admin = Auth.return_user_info(request)
         result = self.post_list(request)
 
         not_run = request.args.get('run', '1') == '0'
 
+        # if we failed to save the execution, we do not continue
         if result[1] >= 300 or not_run:
             return result
-        # To send the absolute url:
-        # url_for(endpoint_name, _external=True)
-        airflow_client = Airflow(current_app.config['AIRFLOW_URL'],
-                                 current_app.config['AIRFLOW_USER'],
-                                 current_app.config['AIRFLOW_PWD'])
-        if not airflow_client.is_alive():
+
+        # We now try to launch the task in airflow
+        af_client = Airflow(**airflow_conf)
+        if not af_client.is_alive():
```

Comment:
Maybe this error message should be different?

Reply:
The client should not need to know that Airflow is behind everything

Reply:
Good point. For debugging is useful but maybe the client should not know this. One option is saying something like "scheduler of jobs is not working, check Airflow is working" or just "job schedule is unavailable." Any other idea is welcome.